sonosco/model.py

```diff
@@ -0,0 +1,168 @@
+import logging
+import torch
+import deprecation
+import inspect
+import torch.nn as nn
+
+from common.class_utils import get_constructor_args, get_class_by_name
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Saver:
+
+    def __init__(self) -> None:
+        super().__init__()
+
+    @deprecation.deprecated(
+        details="This type of saving may cause problems when path of model class changes. Pleas use save_model instead")
+    def save_model_simple(self, model: nn.Module, path: str) -> None:
+        """
+       Simply saves the model using pickle protocol.
```

Comment:
It does need to explicitly mention the difference between this and the following methods